diesel/src/pg/connection/result.rs

```diff
@@ -40,7 +40,8 @@ impl PgResult {
             let count_str = str::from_utf8_unchecked(count_bytes);
             match count_str {
                 "" => 0,
-                _ => count_str.parse().unwrap()
+                _ => count_str.parse()
+                    .expect("Error parsing `rows_affected` as integer value")
```

Comment:
Now that I'm more familiar with the wire protocol, I know why libpq is giving us this as a c string. And it's really dumb.